data/ideas.json

```
@@ -184,5 +184,12 @@
         "description": "An app or website to generate a possible, safest and most acceptable roadmap for the query user seraches about like full stack developement, software developement etc. and also shows the details of each step to gain success",
         "level": "advanced",
         "tags": "roadmap, success, process"
-    }
+    },
+    {
+        "title": "Job Listings Filter",
+        "description": "A Job listing webapp with filter feature for different tags",
+        "level": "intermediate",
+        "tags": "listings, filter, tags"
```

- The first tag is used to match projects with an idea, so please add the book-app tag
- Remove the programming tag, due to is only an idea so the everyone could make on whatever programming language
```suggestion
        "tags": "job-listings-filter, listings, filter, tags"
```

---

Thank you for the suggestion. Will make necessary changes.